src/Component/Symbolizer/FillEditor/FillEditor.tsx

```diff
@@ -218,7 +218,7 @@ export class FillEditor extends React.Component<FillEditorProps> {
                       path: 'FillEditor.fillOpacityField',
                       onChange: this.onFillOpacityChange,
                       propName: 'opacity',
```

Comment:
The propName needs to be changed to `fillOpacity`, too.

```suggestion
                      propName: 'fillOpacity',
```